tools/api-markdown-documenter/src/test/snapshots/markdown/simple-suite-test/sparse-config/test-suite-a/testinterfacewithindexsignature-_indexer_-indexsignature.md

```diff
@@ -2,7 +2,7 @@
 
 Test index signature.
 
-### Signature {#_indexer_-signature}
+### Signature {#\_indexer\_-signature}
```

Comment:
I'm not familiar with this way to specify the anchor, but I'm surprised that we need to escape things inside it... If I understand correctly, the underscores were somehow being interpreted as italicized? Have we tested that linking to the headers by the custom anchor works as expected? Does _that_ need to escape the underscores too?

Reply:
Good questions. Yes, Docusaurus treats the underscores as italics, and as a result, Markdown treats the heading ID in this case as `indexer-signature` instead of `_indexer_-signature`. I have confirmed that this resolves the issue, and it makes sense when you consider how Markdown generates IDs from heading names - it ignores special characters and replaces spaces with `-`s.